graph-db/src/utils/indexing/indexRelatedUrl.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Given a RelatedUrl object, Gremlin connection, and
+ * associated collection, index related URL and build
+ * relationships for any GENERAL DOCUMENTATION that exists
+ * in common with other nodes in the graph database
+ * @param {JSON} relatedUrl a list of RelatedUrls
+ * @param {Connection} gremlin a connection to the gremlin server
+ * @param {Graph Node} dataset the parent collection vertex in the gremlin server
+ * @returns null
+ */
+exports.indexRelatedUrl = (relatedUrl, gremlin, dataset) => {
+  const {
+    Type: type,
+    SubType: subType,
+    URL: url,
+    Description: description,
+  } = relatedUrl
+  if (!!type
+        || !!subType
+        || type !== 'VIEW RELATED INFORMATION'
+        || subType !== 'GENERAL DOCUMENTATION') {
```

Comment:
If subtypes are present there are 25 currently defined documentation subtypes that are not general documentation.  You will filter out a lot of documents that don't have a subtype of GENERAL DOCUMENTATION


ALGORITHM DOCUMENTATION
--
ALGORITHM THEORETICAL BASIS DOCUMENT (ATBD)
ANOMALIES
CASE STUDY
DATA CITATION POLICY
DATA QUALITY
DATA RECIPE
DELIVERABLES CHECKLIST
GENERAL DOCUMENTATION
HOW-TO
IMPORTANT NOTICE
INSTRUMENT/SENSOR CALIBRATION DOCUMENTATION
MICRO ARTICLE
PI DOCUMENTATION
PROCESSING HISTORY
PRODUCT HISTORY
PRODUCT QUALITY ASSESSMENT
PRODUCT USAGE
PRODUCTION HISTORY
PUBLICATIONS
READ-ME
REQUIREMENTS AND DESIGN
SCIENCE DATA PRODUCT SOFTWARE DOCUMENTATION
SCIENCE DATA PRODUCT VALIDATION
USER FEEDBACK PAGE
USER'S GUIDE




ALGORITHM DOCUMENTATION
--
ALGORITHM THEORETICAL BASIS DOCUMENT (ATBD)
ANOMALIES
CASE STUDY
DATA CITATION POLICY
DATA QUALITY
DATA RECIPE
DELIVERABLES CHECKLIST
GENERAL DOCUMENTATION
HOW-TO
IMPORTANT NOTICE
INSTRUMENT/SENSOR CALIBRATION DOCUMENTATION
MICRO ARTICLE
PI DOCUMENTATION
PROCESSING HISTORY
PRODUCT HISTORY
PRODUCT QUALITY ASSESSMENT
PRODUCT USAGE
PRODUCTION HISTORY
PUBLICATIONS
READ-ME
REQUIREMENTS AND DESIGN
SCIENCE DATA PRODUCT SOFTWARE DOCUMENTATION
SCIENCE DATA PRODUCT VALIDATION
USER FEEDBACK PAGE
USER'S GUIDE




ALGORITHM DOCUMENTATION
--
ALGORITHM THEORETICAL BASIS DOCUMENT (ATBD)
ANOMALIES
CASE STUDY
DATA CITATION POLICY
DATA QUALITY
DATA RECIPE
DELIVERABLES CHECKLIST
GENERAL DOCUMENTATION
HOW-TO
IMPORTANT NOTICE
INSTRUMENT/SENSOR CALIBRATION DOCUMENTATION
MICRO ARTICLE
PI DOCUMENTATION
PROCESSING HISTORY
PRODUCT HISTORY
PRODUCT QUALITY ASSESSMENT
PRODUCT USAGE
PRODUCTION HISTORY
PUBLICATIONS
READ-ME
REQUIREMENTS AND DESIGN
SCIENCE DATA PRODUCT SOFTWARE DOCUMENTATION
SCIENCE DATA PRODUCT VALIDATION
USER FEEDBACK PAGE
USER'S GUIDE